lib/modules/datasource/github-runners/index.ts

```diff
@@ -12,9 +12,10 @@ export class GithubRunnersDatasource extends Datasource {
     ubuntu: [{ version: '22.04' }, { version: '20.04' }, { version: '18.04' }],
     macos: [
       { version: '13' },
-      { version: '13-xl' },
+      { version: '13-large' },
```

Comment:
Do I need to wait until this PR is merged?

- https://github.com/renovatebot/renovate/pull/27241